### DISCUSSION TITLE
Cluster Joining in RabbitMQ

### DISCUSSION THREAD

**Discussion by kmuthugtk** [rabbitmq/rabbitmq-server#11563] [Other] [answered]
### Describe the bug

I am trying to join Node 1 and Node 2 to the cluster. Could anybody please help me resolve this issue?

![Untitled Diagram-Page-10 drawio](https://github.com/rabbitmq/rabbitmq-server/assets/76728951/3691d468-94d9-4bda-83c2-b1cd18bb397d)

### Reproduction steps

1.
2.
3.
...

### Expected behavior

No

**Comment by michaelklishin** [comment, marked as answer]
The message means that target node cannot connect to the `[redacted]01` node to determine if their [feature flags](https://www.rabbitmq.com/docs/feature-flags) are available.

There can be any number of reasons for that, see `[redacted]01`'s logs for clues as well as

 * [Clustering: how nodes authenticate](https://www.rabbitmq.com/docs/clustering#erlang-cookie) to each toher
 * [Clustering: troubleshooting](https://www.rabbitmq.com/docs/clustering#cookie-file-troubleshooting)
 * [Networking: ports](https://www.rabbitmq.com/docs/networking#ports)
 * [Networking: inter-node communication port range](https://www.rabbitmq.com/docs/networking#epmd-inet-dist-port-range)
 * [Troubleshooting Networking](https://www.rabbitmq.com/docs/troubleshooting-networking)
 * If any of the nodes is configured to use [TLS for inter-node communication](https://www.rabbitmq.com/docs/clustering-ssl), it must be enabled for all nodes and CLI tools
